src/App.js

```diff
@@ -58,7 +60,7 @@ function App() {
    */
 
   useEffect(() => {
-    const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=${process.env.REACT_APP_APIKEY}`;
+    const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&units=metric&appid=48a50678eba83ab8e75543e3bf60a915`;
```

Comment:
Why do we have the credentials hardcoded? We should really avoid this.

Reply:
yeah updated